compiler/src/main/java/org/robovm/compiler/util/ToolchainUtil.java

```diff
@@ -141,11 +152,69 @@ public static void pngcrush(Config config, File inFile, File outFile) throws IOE
     }
 
     public static void textureatlas(Config config, File inDir, File outDir) throws IOException {
+        List<String> opts = new ArrayList<String>();
+        TextureAtlas atlasConfig = config.getTools().getTextureAtlas();
+        int outputFormat = 1;
+        int maxTextureDimension = 1;
+        
+        if (atlasConfig != null) {
+            outputFormat = 1 + atlasConfig.getOutputFormat().ordinal();
+            maxTextureDimension = 1 + atlasConfig.getMaximumTextureDimension().ordinal();
+            
+            if (atlasConfig.usePowerOfTwo()) {
+                opts.add("-p");
+            }
+        }
+        
         new Executor(config.getLogger(), getTextureAtlas())
-            .args(inDir, outDir)
+            .args(opts, "-f", outputFormat, "-s", maxTextureDimension, inDir, outDir)
             .exec();
     }
     
+    public static void actool(Config config, File inDir, File outDir) throws IOException {
+        List<Object> opts = new ArrayList<>();
+
+        String appIconSetName = null;
+        String launchImagesName = null;
+        
+        final String appiconset = "appiconset";
+        final String launchimage = "launchimage";
+        
+        for (String fileName : inDir.list()) {
+            String ext = FilenameUtils.getExtension(fileName);
+            if (ext.equals(appiconset)) {
+                appIconSetName = FilenameUtils.getBaseName(fileName);
+            } else if (ext.equals(launchimage)) {
+                launchImagesName = FilenameUtils.getBaseName(fileName);
+            }
+        }
+        if (appIconSetName != null || launchImagesName != null) {
+            if (appIconSetName != null) {
+                opts.add("--app-icon");
+                opts.add(appIconSetName);
+            }
+            if (launchImagesName != null) {
+                opts.add("--launch-image");
+                opts.add(launchImagesName);
+            }
+
+            File partialInfoPlist = new File(inDir, "assetcatalog-partial-info.plist");
+            opts.add("--output-partial-info-plist");
+            opts.add(partialInfoPlist);
+        }
+        
+        opts.add("--platform");
+        if (IOSTarget.isDeviceArch(config.getArch())) {
+            opts.add("iphoneos");
+        } else if (IOSTarget.isSimulatorArch(config.getArch())) {
+            opts.add("iphonesimulator");
+        }
+        
+        new Executor(config.getLogger(), getACTool())
+            .args(inDir, opts, "--output-format", "human-readable-text", 
+                    "--minimum-deployment-target", "6.0", "--compile", outDir).exec();
```

Comment:
Does Xcode also hardcode 6.0 here or is this something we need to be able to pass in based on the MinimumOSVersion specified in the user's Info.plist.xml?